pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -19,13 +19,41 @@ package cpumanager
 import (
 	"fmt"
 	"strconv"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
 )
 
 const (
 	// FullPCPUsOnlyOption is the name of the CPU Manager policy option
 	FullPCPUsOnlyOption string = "full-pcpus-only"
 )
 
+var (
+	stableOptions = sets.NewString()
+	betaOptions   = sets.NewString(
+		FullPCPUsOnlyOption,
+	)
+	alphaOptions = sets.NewString()
+)
+
+func IsPolicyOptionAvailable(option string) error {
```

Comment:
I think this function should be called `Assert...` or `Check...` rather than `Is...`. If it begins with `Is..` I would expect it to return a bool, not an error (which then makes it hard to customize the error returned).

Reply:
Right, good point. `Check` it is.